### DISCUSSION TITLE
Nebula graph error

### DISCUSSION THREAD

**Discussion by amrsaad7** [vesoft-inc/nebula#5804] [Q&A] [answered]
I am trying to open nebula with docker on a macbook, I face this error whenever i try to create a new space "Host not enough".
any solutions?

**Comment by amrsaad7** [comment]
<img width="1429" alt="Screenshot 2024-01-11 at 16 09 26" src="https://github.com/vesoft-inc/nebula/assets/56649390/34e6488d-796f-4b7d-9018-743644b4ccbc">

**Reply by wey-gu** [reply, marked as answer]
Author of the docker extension here, sorry this shouldn't happen if everything was implemented perfectly by me.

In this case we have to activate storage host(by add hosts command.

Kindly check get started, I added that manual step there for you, you should copy it and run it in studio console :)!

Welcome to the community!

**Reply by amrsaad7** [reply]
It works now, thank you.

**Reply by amrsaad7** [reply]
I have couple of questions regarding this product. can i ask you here or you prefer to talk in private?

**Reply by wey-gu** [reply]
Both will do, for private chat if you prefer a private chat, ping me via the slack(we could schedule meeting as need there), or we could have async public discussion here as you feel comfortable:)

Sorry for the late response.
